src/components/TodoItemLoader/TodoItemLoader.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo,
+};
+
+export const TodoItemLoader: React.FC<Props> = ({ todo }) => {
+  return (
+    <div className="todo">
+      <label className="todo__status-label">
+        <input type="checkbox" className="todo__status" />
+      </label>
+
+      <span className="todo__title">{todo.title}</span>
+      <button type="button" className="todo__remove">×</button>
+
+      <div className="modal overlay is-active">
```

Comment:
Consider moving the modal to a separate component for better separation of concerns and reusability. 🔄

Reply:
Not resolved. Just create a component for the modal.